src/mesh/region.cc

```diff
@@ -234,14 +240,19 @@ void RegionDB::close() {
     // set default sets
    for(unsigned int i=0; i< size(); i++) {
        Region reg(i, *this);
+   	   RegionSet region_set;
+   	   region_set.push_back( reg );
+
 
        if (reg.is_boundary() && (reg.boundary_idx() < boundary_size()) ) {
            add_to_set("BOUNDARY", reg );
            add_to_set("ALL", reg);
+       	   add_set(reg.label(), region_set);
```

Comment:
Problem with earlier added regions is with regions renamed by user. In this case we would must rename appropriate set too.